test/relaxed-simd.wast.wasm.fromBinary

```diff
@@ -0,0 +1,113 @@
+(module
```

Comment:
I don't think we need to add a binary test for this, doing binary roundtripping as part of a lit test should be enough.

Reply:
I'm not super sure what all these files do, can all the fromBinary files be removed? there are also test/relaxed-simd.wast.fromBinary and test/relaxed-simd.wast.fromBinary.noDebugInfo

Reply:
With the lit test, I removed all these files.